### ISSUE TITLE
auth 403

### ISSUE THREAD

**Issue by huhuegg** [ClaudiuGeorgiu/PlaystoreDownloader#21]
https://android.clients.google.com/auth

**Comment by b-reynolds** [comment]
I'm also getting a 403 error when attempting to request an authentication token... the response body contains a captcha 😞 

`
Error=CaptchaRequired
CaptchaToken={TOKEN}
CaptchaUrl=https://accounts.google.com/Captcha?ctoken={TOKEN}&v=2
`

**Comment by ClaudiuGeorgiu** [comment]
Hi, can you try opening that link in the browser and solving the captcha? Maybe that will "unlock" your account.

**Comment by b-reynolds** [comment]
The link leads me to the captcha image but it doesn't provide a submission form or anything for submitting the solution.

I'm trying to figure out how to submit the solution to see if that helps. https://developers.google.com/recaptcha/docs/verify looked promising but I'm getting this response when sending the token as `secret` and captcha as `response`.

`
{ "success": false, "error-codes": [ "invalid-input-response", "invalid-input-secret" ] }
`

response

**Comment by b-reynolds** [comment]
I'm probably using the wrong endpoint or sending a bad request.. struggling to find anything much when searching for https://accounts.google.com/Captcha?ctoken

**Comment by hoffie** [comment]
I have analyzed a similar issue in another project which uses the Android Accounts API (Android App Aurora Store). My results so far are that Google seems to decide whether the require a Captcha by analyzing TLS protocol details. More specifically, I was able to make it work by forcing specific ciphers and disabling ALPN.

I have not been successful with solving the Captcha and submitting it anywhere. My solution was to avoid triggering the Captcha.

I am detailing my results here: https://gitlab.com/AuroraOSS/AuroraStore/issues/217#note_249390026

See also NoMore201/googleplay-api#98
